package exceptionhandling;

public class ExceptionDemo {
	
	private void checkArithmaticException() {
		
		int a = 0;
		int  result = 10/a;
		System.out.println("result :" +a);
		
	
	}
	
	public static void main (String [] args) {
		
		ExceptionDemo demo = new ExceptionDemo ();
		
		
	}
	
	
	

}
